app/src/pages/inside/settingsPage/settingTabs/notificationsTab/forms/forms.scss

```diff
@@ -0,0 +1,35 @@
+.form-input {
+  width: 310px;
+  position: relative;
+}
+
+.form-input-owner {
+  margin-bottom: 20px;
+  padding-left: 170px;
+}
+
+.dropdown {
+  border: 1px solid #cccccc;
```

Comment:
It's not allowed to define colors directly, without of using color-variables. Additionally, these css-selector and it's styles no needed at all. Now I see double borders of dropdowns on that form.

Reply:
removed